frontend/packages/console-shared/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -96,7 +96,7 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
                   className="odc-quick-search-list__item-content"
                   dataListCells={[
                     <DataListCell isIcon key={`${item.uid}-icon`}>
-                      {getIcon(item)}
+                      {item?.icon?.node ? item.icon.node : getIcon(item)}
```

Comment:
nit: cane we have as below
```suggestion
                      {item.icon?.node ?? getIcon(item)}
```